Week4/overview.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objects, Classes, new, this and prototypes\n",
+    "\n",
+    "> Created by the one and only Jim, you can find him on our [Slack](https://hackyourfuture.slack.com/team/U383PTTK9) and on [GitHub](https://github.com/remarcmij)!\n",
+    "\n",
+    "> This is a Jupyter notebook, you can run the code immediately if you do some setting up steps, although it is not necessary to do so as we have provided the results. If you want to set it up you can go through the following [article](https://www.tomche.space/post/using-javascript-kernel-in-vscode-jupyter-notebooks/)\n",
```

Comment:
Not sure if it is worth the trouble for students to install all the prerequisites for using Notebooks with JS. They will not need it for the rest of the curriculum and they may waste time and energy getting this working. I would suggest to treat this as read-only for them.

Reply:
Good point, will reword it to tell them not to do it as I think vscode might try to get them to once it identifies ipynb file name